### PR TITLE
Urban burn granting

### PR DESCRIPTION
**Pull request by aerugo**

### CODE REVIEW

File: lib/tasks/refill_grants.rake [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# to import run bundle exec rake refill_grants
```
Comment (rootux) [inline lib/tasks/refill_grants.rake:1]: @aerugo Can you explain about this and why it is needed. I'm curious.

File: app/views/camps/show.html.erb [hunk 2]
```diff
@@ -316,16 +321,13 @@
...
-                      <% if current_user&.admin %>
-                        <%= button_to t(:delete_this_creation), { action: 'destroy' },
-                                      { method: :delete, class: "btn btn-danger", data: { confirm: t(:delete_are_you_sure) } } %>
-                      <% end %>
-
-                      <br>
-                      <%= t("activerecord.models.user.one") %> <%= @camp.creator.email %>
                     <% end %>
 
+                  <% if current_user && (@camp.creator == current_user || current_user.admin) %>
```
Comment (rootux) [inline app/views/camps/show.html.erb:326]: @aerugo One thing to notice that if user has granted this and then a project gets deleted - all the grants are currently get "lost"

**Comment by aerugo** [comment]
Hey! I didn't mean to pull request this to Metaburn. I closed the pull
request. This feature is only for the Urban Burn repo, and it's a rake task
that can be scheduled to keep refilling user grants until grants have run
out. This is to encourage granting and trust those who make decisions with
more decision making. It's a bit of an experiment, we'll see how it goes.

**Comment by rootux** [comment]
I love this experiment.
great job man.

**Pull request closed without merging**